command/cli.go

```diff
@@ -98,7 +98,7 @@ func (cli *CLI) Run(args []string) int {
 
 	// Development only flags. Not printed with -h, -help
 	f.StringVar(&clientType, "client-type", "", "Use only when developing"+
-		"consul-terraform-sync binary. Defaults to Terraform client if empty or"+
+		" consul-terraform-sync binary. Defaults to Terraform client if empty or"+
 		"unknown value. Values can also be 'development' or 'test'.")
```

Comment:
```suggestion
	f.StringVar(&clientType, "client-type", "", "Use only when developing "+
		"consul-terraform-sync binary. Defaults to Terraform client if empty or "+
		"unknown value. Values can also be 'development' or 'test'.")
```

Believe there is also a missing space between `orunknown`